packages/slate-react/src/utils/scroll-to-selection.js

```diff
@@ -75,11 +77,16 @@ function scrollToSelection(selection) {
     height = offsetHeight
     yOffset = scrollTop
```

Comment:
might want to assign do it as
```
scollerRect = scroller.getBoundingClientRect()
yOffset = scrollTop - scrollerRect.top
xOffset = scrollLeft - scrollerRect.left
```
since `scrollerYOffset` and `scrollerXOffset` is only being used when `scroller != window`